src/main/java/uk/gov/digital/ho/hocs/client/casework/dto/CreateMigrationCaseRequest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.digital.ho.hocs.client.casework.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import uk.gov.digital.ho.hocs.client.workflow.dto.DocumentSummary;
+
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@EqualsAndHashCode
```

Comment:
While good practice to have anyway - this default behaviour would likely never have an impact as it will use all of the fields. Is there much use to that?